### ISSUE TITLE
Just execute this file

### ISSUE THREAD

**Issue by ghanan** [undergroundwires/privacy.sexy#240]
So, this goes of privacy and security, but in Linux you expect we to execute a file just downloaded. Good job!

**Comment by undergroundwires** [comment]
Hi @ghanan , if you were to be constructive, what improvement would you like to see?

**Comment by undergroundwires** [comment]
Are you not able to download?

1. You can download and check with download button.
2. You can copy the contents with copy button and create file yourself.
3. You see live view of script you create on right.

![image](https://github.com/undergroundwires/privacy.sexy/assets/15555035/4983b63e-d6e0-45f0-9b0a-4e9dce5499d2)

**Comment by ghanan** [comment]
Yes, now I can view them. When I choose linux I found an appimage to download and that made me think it was the way to execute the scrips. Excuse me.
I see you have removed the appimage, now the link is broken (https://github.com/undergroundwires/pr.vacy.sexy/releases/download/0.11.2/privacy.sexy-0.11.2.AppImage)

**Comment by undergroundwires** [comment]
Ah I see, you interpret buttons on bottom left like they're the ones that lets you execute the script. How about we change the UI on bottom left, so it's more clear button like `Install desktop version` and once clicked it shows a dialog modal with installation instructions and what desktop distribution does (allows offline usage). Maybe this can give more clarity, or let me know if you have another idea.

Your link has typo `pr.vacy` instead of `privacy` this link should work: https://github.com/undergroundwires/privacy.sexy/releases/download/0.11.2/privacy.sexy-0.11.2.AppImage. But if you want to run AppImage, get the latest version, this is an old release that doesn't include Linux scripts.

**Comment by ghanan** [comment]
I think your solution would be enough.
Thanks.
